public/v1/components/schemas.yaml

```diff
@@ -1294,6 +1296,14 @@ components:
           type: boolean
           description: |
             Indicates whether a trusted authority signed the certificate.
+        protocol:
+          type: string
+          description: |
+            The minimum TLS protocol version supported by this cipher suite.
```

Comment:
Ah, I see I got this wrong. The goal was to show the actual protocol used, so we should update the probe to report `.getProtocol()` instead.